layouts/shortcodes/comprehensive-pentest.md

```diff
@@ -0,0 +1 @@
+A Comprehensive Pentest is performed for [security audit](/getting-started/glossary/#security-audit), [compliance audit](/getting-started/glossary/#compliance-audit), or customer attestation and includes comprehensive [reports](/platform-deep-dive/pentests/reports/) intended for external stakeholders
```

Comment:
@yogipetkar Do you approve this definition of Comprehensive Pentest?

Reply:
@ana-dashuk-cobalt - Please look at slide #7 in https://docs.google.com/presentation/d/1Rujaf_XSwW5VdiSHrYuecNjAcrmobx97jYI5SM9NLrg/edit?usp=sharing for the definition. It matches what you have captured.